config/initializers/figaro.rb

```diff
@@ -0,0 +1,2 @@
+Figaro.require_keys("GITHUB_KEY", "GITHUB_SECRET",
+                    "glassdoor_partner_id", "glassdoor_key")
```

Comment:
This will proactively raise errors. Explained in the [Figaro docs](https://github.com/laserlemon/figaro): ` If any of the configuration keys above are not set, your application will raise an error during initialization. This method is preferred because it prevents runtime errors in a production application due to improper configuration.`